### PR TITLE
Release v4.4.0

### PR DESCRIPTION
**Pull request by tscheypidi**
Release of MAgPIE in version 4.4.0

### CODE REVIEW

**Comment by bodirsky** [comment]
Has been reviewed extensively before

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,94 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 
+## [4.4.0] - 2021-12-13
+
+### changed
+- **inputs** new default LPJmL version with growing season adaptation (gsadapt) on 
+- **51_nitrogen** parameter change in rescaled_jan21, now including regionalized climate-dependent leaching factors
+- **config** Update default configuration to new input data (especially cellular inputs) including all module realization updates (14_yield, 22_processing, 30_crop, 38_factor_costs, 39_landconversion). Moreover, climate impatcs (cc options for biophysical inputs) are activiated as default. New best_calib calibration routine is activated as default.
```
Comment (FelicitasBeier) [inline CHANGELOG.md:14]: typo: impatcs -> impacts

**Review by abhimishr — APPROVED** [review]
🚀

**Review by bodirsky — APPROVED** [review]

**Review by flohump — APPROVED** [review]
Model code and outcomes have been reviewed extensively before

File: modules/10_land/feb15/declarations.gms [hunk 2]
```diff
@@ -18,7 +18,7 @@ variables
 positive variables
  vm_land(j,land)                  Land area of the different land types (mio. ha)
  vm_landexpansion(j,land)         Land expansion (mio. ha)
- v10_landreduction(j,land)        Land reduction (mio. ha)
+ vm_landreduction(j,land)        Land reduction (mio. ha)
```
Comment (FelicitasBeier) [inline modules/10_land/feb15/declarations.gms:21]: space missing to look tidy

File: modules/10_land/landmatrix_dec18/declarations.gms [hunk 3]
```diff
@@ -17,11 +17,13 @@ variables
 positive variables
  vm_land(j,land)                             Land area of the different land types (mio. ha)
  vm_landexpansion(j,land)                    Land expansion (mio. ha)
- v10_landreduction(j,land)                   Land reduction (mio. ha)
+ vm_landreduction(j,land)                   Land reduction (mio. ha)
```
Comment (FelicitasBeier) [inline modules/10_land/landmatrix_dec18/declarations.gms:20]: spacing of description

File: modules/10_land/landmatrix_dec18/equations.gms [hunk 4]
```diff
@@ -9,9 +9,16 @@
 
 *' The following three equations describe the general structure of the land transition matrix.
 *' The first equation defines the total amount of land to be constant over time.
+*' The two balancing variables `v10_balance_positive` and `v10_balance_negative` are needed
```
Comment (FelicitasBeier) [inline modules/10_land/landmatrix_dec18/equations.gms:12]: Are they reported somewhere such that one can check whether this is the case that they only deviate from zero in exceptional cases? Is there a possibility of including this as an automated test at the end of the run?

**Pull request merged**